cni/network/multitenancy.go

```diff
@@ -249,13 +255,14 @@ func (m *Multitenancy) getNetworkContainersInternal(
 		return nil, []net.IPNet{}, fmt.Errorf("%w", err)
 	}
 
-	log.Printf("Network config received from cns %+v", ncConfigs)
+	log.Logger.Info("Network config received from cns", zap.Any("nconfig", ncConfigs))
 
 	subnetPrefixes := []net.IPNet{}
 	for i := 0; i < len(ncConfigs); i++ {
 		subnetPrefix := m.netioshim.GetInterfaceSubnetWithSpecificIP(ncConfigs[i].PrimaryInterfaceIdentifier)
 		if subnetPrefix == nil {
-			log.Printf("%w %s", errIfaceNotFound, ncConfigs[i].PrimaryInterfaceIdentifier)
+			log.Logger.Error(errIfaceNotFound.Error(),
+				zap.String("interface", ncConfigs[i].PrimaryInterfaceIdentifier))
```

Comment:
let's name this as nodeIP instead of interface

Reply:
done